TodoApp/test/TodoApp.Api.Tests/Controllers/ItemsController.test.cs

```diff
@@ -118,8 +115,13 @@ private static IItemRepository MockItemRepository()
         public async Task PutItem_ExistingItem_ItemUpdated()
         {
             var id = Guid.Parse("c5cc89a0-ab8d-4328-9000-3da679ec02d3");
+            var updateItem = new Item { Id = id, Text = "Make a coffee" };
 
-            var updateItem = new Item {Id = id, Text = "Make a coffee"};
+            _repository
+                .Update(id,
+                        Arg.Is<Item>(item => item.Id.ToString() == "c5cc89a0-ab8d-4328-9000-3da679ec02d3" &&
```

Comment:
If you compare two `item` you should always use comparer so you have the comparison logic at one place.

Reply:
Moreover, this call is really long - always introduce a method in case your lambdas grow long or complex.